src/components/lifetime/PriceTable.tsx

```diff
@@ -150,7 +150,7 @@ const PriceTable = ({
                       cta={['checkout', product.priceId]}
                       lang={lang}
                       billingFrequency={Interval.Lifetime}
-                      popular={lifetimeMode === 'normal' ? product.storage === '10TB' : product.storage === '5TB'}
+                      popular={lifetimeMode === 'celebration' ? product.storage === '10TB' : product.storage === '5TB'}
```

Comment:
Be careful with this change. This component is used across all the lifetime pages we have. Is this the desired behavior? Should that card be for celebration pages and not for normal pages? If not, you can do something like:
`lifetimeMode === 'celebration' || lifetimeMode === 'normal'`

Or you can add an array with the pages that should change the popular card to the 10TB plan and then use the includes method:

````

const LIFETIME_MODES_WITH_POPULAR_10TB = ['celebration', 'normal'];

const popularStoragePlan = LIFETIME_MODES_WITH_POPULAR_10TB.includes(lifetimeMode) 
  ? '10TB' 
  : '5TB';

...
popular={product.storage === popularStoragePlan}

`````